apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -72,9 +80,18 @@ Page {
                     }
                     MenuItem {
                         readonly property string site: removeProtocolTypeFromUri(homePageConfig.value)
-                        //: Instead of %1 site address will be displayed
-                        //% "Custom website %1"
-                        property string title: qsTrId("sailfish_browser-me-home_page_custom").arg(homePageConfig.value === "about:blank" ? "" : site)
+
+                        property string title: {
+                            if (homePage._homePageBlank || site.trim() === "") {
+                                //: Shown when site is empty
+                                //% "Custom website"
+                                return qsTrId("sailfish_browser-me-home_page_custom_empty")
+                            } else {
+                                //: Instead of %1 site address will be displayed
+                                //% "Custom website %1"
```

Comment:
There was some discussion about changing the wording for these (taken from the notes):
 1. "Default home page" can be renamed to the "Start view"
 2. "Custom website" change "Web page" also update the actual dialog

But I think it was decided that these changes would be handled separately, so I'm just noting it here for reference.

Reply:
Yes, let's create a separate MR for this.